### DISCUSSION TITLE
LaTex Math highlighing

### DISCUSSION THREAD

**Discussion by skwde** [quarto-dev/quarto-cli#11221] [Q&A]
This is basically the same as this issue
 
- <https://github.com/quarto-dev/quarto-vscode/issues/29>

which is read only now.

In any case, it is still not clear (to me) how one should use `align` in quarto documents.

```qmd
---
title: Display math highlighting example
format:
  html:
    html-math-method: katex
  pdf:
    documentclass: article
    keep-tex: true
---
Here is some display math using bare `\begin{align} ... \end{align}`:
\begin{align}
C &= \max_{p_X}{I(X;Y)}\\
  &= \max_{p_X}{H(X) - H(X\mid Y)}
\end{align}
It is _not_ highlighted.

Here is some display math using `$$\begin{align} ... \end{align}$$`:

$$
\begin{align}
C &= \max_{p_X}{I(X;Y)}\\
  &= \max_{p_X}{H(X) - H(X\mid Y)}
\end{align}
$$
It _is_ highlighted, but causes issues in pdf/latex output.
```

The first `align` works but is not highlighted. 
While the second triggers an error but is highlighted.

**Comment by bhavitsharma** [comment]
is there any workaround for it? i really like the preview mode the vscode's extension but that breaks rendering to PDF
